src/data/svgs.ts

```diff
@@ -2824,17 +2824,24 @@ export const svgs: iSVG[] = [
     url: 'https://postcss.org/'
   },
   {
-    title: "SVG",
-    category: ["Language", "Design"],
-    route: "/library/svg.svg",
-    wordmark: "/library/svg_wordmark.svg",
-    url: "https://www.w3.org/TR/SVG/"
+    title: 'SVG',
+    category: ['Language', 'Design'],
+    route: '/library/svg.svg',
+    wordmark: '/library/svg_wordmark.svg',
+    url: 'https://www.w3.org/TR/SVG/'
```

Comment:
Fix incorrect quote formatting (based on [the `.prettierrc`](https://github.com/pheralb/svgl/blob/main/.prettierrc))